torch/compiler/__init__.py

```diff
@@ -28,6 +28,8 @@
     "is_exporting",
     "save_cache_artifacts",
     "load_cache_artifacts",
+    "skip_guard_on_inbuilt_nn_modules_unsafe",
+    "skip_guard_on_all_nn_modules_unsafe",
```

Comment:
Will the other guard filters be added too?

Reply:
Done